src/css/game.css

```diff
@@ -9,8 +9,8 @@
 }
 
 #canvas1 {
-  width: 83%;
-  position: absolute;
+  width: 100%;
+  position: static;

```

Comment:
Might have deleted the whole `position: static;` as well then, this is the default value

Reply:
Done, thanks!